README.rst

```diff
@@ -47,6 +47,11 @@ download them and check the loading directory is correct in the
 notebook. By default we assume you have downloaded the data in the
 ``./data/`` directory in the repository root.
 
+For the **hippocampus dataset** the data is automatically downloaded when you call 
+``cebra.datasets.init()``. The default path is the ``./data`` folder
+in the repository root. It will be possible to automatically download the remaining
+datasets soon.
+
 For different paths, you can specify the ``CEBRA_DATADIR=...``
 environment variable. You can do this by placing
 ``import os; os.environ['CEBRA_DATADIR'] = "path/to/your/data"`` at the
```

Comment:
Can we add the code snippet from @MMathisLab diff here (#3 ) to the README here as well? We can remove it again once all datasets support auto-download.